example/spec/parallel/kinesisTests/KinesisTestTriggerSpec.js

```diff
@@ -106,7 +106,6 @@ describe('The Cloud Notification Mechanism Kinesis workflow', () => {
 
     await deleteExecution({ prefix: testConfig.stackName, executionArn: failingWorkflowExecution.executionArn });
     await deleteExecution({ prefix: testConfig.stackName, executionArn: workflowExecution.executionArn });
-    await removeFromCMR({ prefix: testConfig.stackName, granuleId });
     await deleteGranule({ prefix: testConfig.stackName, granuleId });
```

Comment:
Should we pull out the CMR step from this workflow, given it's only being used here?

Reply:
I'm not sure what you mean. The reason for pulling the `removeFromCMR` call is that there are two worfklows run by this test: a success and failing workflow. After the second workflow, the workflow is no longer published, so it's not necessary to remove it from CMR before deleting it